src/requirements.cpp

```diff
@@ -104,8 +104,12 @@ std::string tool_comp::to_string( int batch ) const
 std::string item_comp::to_string( int batch ) const
 {
     const int c = std::abs( count ) * batch;
+    auto type_ptr = item::find_type( type );
+    if( type_ptr->stackable ) {
+        return string_format( ( "%s (%d)" ), type_ptr->nname( 1 ).c_str(), c );
```

Comment:
Excessive brackets around the string literal. Also: `type_ptr` could be `const`.